### PR TITLE
SUBMARINE-579. [WEB] Add duration information to experiment

### PR DESCRIPTION
**Pull request by kobe860219**
### What is this PR for?
* Implement duration of experiment.
* Remove progress bar.

### What type of PR is it?
[Feature]

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-579

### How should this be tested?
https://travis-ci.org/github/kobe860219/submarine/builds/713190854

### Screenshots (if appropriate)
Submarine-Workbench_Experiment_Update

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by kobe860219** [comment]
@pingsutw  @liuxunorg  Please help me review this pr. Thanks :))

File: submarine-workbench/workbench-web-ng/src/app/pages/workbench/experiment/experiment-info/experiment-info.component.ts [hunk 1]
```diff
@@ -116,6 +127,31 @@ export class ExperimentInfoComponent implements OnInit {
     );
   }
 
+  durationHandle(secs: number) {
+    var hr = Math.floor(secs / 3600);
+    var min = Math.floor((secs - hr * 3600) / 60);
+    var sec = Math.round(secs) - hr * 3600 - min * 60;
+    var showHr;
+    var showMin;
+    var showSec;
```
Comment (pingsutw) [inline submarine-workbench/workbench-web-ng/src/app/pages/workbench/experiment/experiment-info/experiment-info.component.ts:136]:
```suggestion
    const hr = Math.floor(secs / 3600);
    const min = Math.floor((secs - hr * 3600) / 60);
    const sec = Math.round(secs) - hr * 3600 - min * 60;
    let showHr;
    let showMin;
    let showSec;
```

Comment (kobe860219) [reply submarine-workbench/workbench-web-ng/src/app/pages/workbench/experiment/experiment-info/experiment-info.component.ts]: @pingsutw  Thanks your suggestion. I have updated this pr.

**Comment by kobe860219** [comment]
@jiwq Sorry about that I forgot to update screenshots. I have update it :)

**Comment by jiwq** [comment]
Thanks @kobe860219 for the contribution, @liuxunorg and @pingsutw help to review.

**Pull request closed without merging**